schematic/store/synapse.py

```diff
@@ -2231,14 +2233,16 @@ def upsertTable(self, sg: SchemaGenerator,):
         except(SynapseHTTPError) as ex:
             # If error is raised because Table has old `Uuid` column and not new `Id` column, then handle and re-attempt upload
             if 'Id is not a valid column name or id' in str(ex):
+                print('triggering this function call that I want to test')
                 self._update_table_uuid_column(sg)
                 synapseDB.upsert_table_rows(table_name=self.tableName, data=self.tableToLoad)
             # Raise if other error
             else:
                 raise ex
 
         return self.existingTableId
-
+    
+    @profile(sort_by='cumulative', strip_dirs=True)
```

Comment:
will be removed after other people confirm that performance is the same as PR: https://github.com/Sage-Bionetworks/schematic/pull/1229